src/HDInsight/HDInsight/help/Add-AzHDInsightClusterIdentity.md

```diff
@@ -60,11 +61,12 @@ PS C:\> New-AzHDInsightClusterConfig `
             | Add-AzHDInsightClusterIdentity `
                 -AadTenantId $tenantId `
                 -ObjectId $objectId `
+                -Application $applicationId
                 -CertificateFilePath $certificateFilePath `
                 -CertificatePassword $certificatePassword `
             | New-AzHDInsightCluster `
                 -ClusterType Hadoop `
-                -OSType Windows `
+                -OSType Linux `
```

Comment:
You still support Windows, right?

Reply:
> You still support Windows, right?

not any more. Customer can't create cluster with Windows now.

Reply:
> You still support Windows, right?

And after this PR is merged, we will submit a pr including breaking change from az.hdinsight-preview to master.  We want these prs can be included in the Ignite release.

Reply:
Sure. To confirm this, are the breaking change already merged in to az.hdinsight-preview branch and also include the changelog. Am I right?

Reply:
> Sure. To confirm this, are the breaking change already merged in to az.hdinsight-preview branch and also include the changelog. Am I right?

Yes.

Reply:
> > Sure. To confirm this, are the breaking change already merged in to az.hdinsight-preview branch and also include the changelog. Am I right?
> 
> Yes. But when submit a pr from az.hdinsight-preview to master we need to solve conflicts by ourselves, right? If So we hope this pr can be merged at today so that we can have enough time to solve conflicts.

Reply:
Sure